docs.html

```diff
@@ -174,6 +173,78 @@ <h3>Copy and Use it ⬇</h3>
 
                 <hr>
 
+                <section id="iconButton">
+
+                    <h2>Icons Buttons</h2>
+
+                        <div class="componentDiv">
+
```

Comment:
I feel the design would look cleaner without the borders for each component.

Reply:
Done ! looks much cleaner